client/api/smb/v1alpha1/api.pb.go

```diff
@@ -0,0 +1,370 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: client/api/smb/v1alpha1/api.proto
```

Comment:
nit: the path appears different from: https://github.com/kubernetes-csi/csi-proxy/blob/master/client/api/disk/v1alpha1/api.pb.go#L2 and https://github.com/kubernetes-csi/csi-proxy/blob/master/client/api/volume/v1alpha1/api.pb.go#L2 . Can you fix up the path please?